plutus-core-interpreter/test/DynamicBuiltins/Definition.hs

```diff
@@ -55,8 +54,8 @@ test_dynamicFactorial :: TestTree
 test_dynamicFactorial =
     testCase "dynamicFactorial" $ do
         let env = insertDynamicBuiltinNameDefinition dynamicFactorialDefinition mempty
-            lhs = typecheckEvaluateCek env $ applyFactorial dynamicFactorial 3 10
-            rhs = typecheckEvaluateCek mempty $ applyFactorial factorial 3 10
+            lhs = typecheckEvaluateCek env $ applyFactorial dynamicFactorial 10
+            rhs = typecheckEvaluateCek mempty $ applyFactorial factorial 10
```

Comment:
The point of `applyFactorial` was that it encapsulated some size logic. I guess it doesn't do too much now and we can remove this function and use `Apply` instead.